pmd-doc/pom.xml

```diff
@@ -24,7 +24,7 @@
                     <plugin>
                         <groupId>org.codehaus.mojo</groupId>
                         <artifactId>exec-maven-plugin</artifactId>
-                        <version>1.6.0</version>
+                        <version>3.0.0</version>
```

Comment:
Let's define the version in the main parent pom under "pluginManagment" and remove the explicit version here in pmd-doc

Reply:
I thought about it but forgot and did just find/replace. Will fix